core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -60,7 +60,8 @@ class ExecutorSummary private[spark](
     val totalShuffleRead: Long,
     val totalShuffleWrite: Long,
     val maxMemory: Long,
-    val executorLogs: Map[String, String])
+    val executorLogs: Map[String, String],
+    val isRemoved: Boolean)
```

Comment:
the tests are failing b/c this is not binary compatible, but I think this is OK.  In particular, from http://spark.apache.org/docs/latest/monitoring.html
we say:

> New fields may be added to existing endpoints

So we should add a mima exclusion [here](https://github.com/apache/spark/blob/master/project/MimaExcludes.scala#L37).  The logs of the PR builder tell you exactly what to add:
`ProblemFilters.exclude[MissingMethodProblem]("org.apache.spark.status.api.v1.ExecutorSummary.this")`